README.md

```diff
@@ -26,6 +34,6 @@ yarn global add cats-cli
 
 ## Command Line
 
-Commands | Desciption | Options
---- | --- | ---
-```cats initialize``` | Create in the current directory a brand new project | _None_
\ No newline at end of file
+| Commands          | Desciption                                          | Options |
+| ----------------- | --------------------------------------------------- | ------- |
```

Comment:
Why ? Auto table markdown already handle this. It adds complexity for the next edition ?

Reply:
🤷‍♂️ prettier does what prettier does ? :D

Reply:
If the difficulty is handled by prettier, no problem then